src/main/resources/jp/ikedam/jenkins/plugins/jobcopy_builder/Messages_ja.properties

```diff
@@ -28,6 +28,8 @@ AdditionalFileset.DisplayName=\u8ffd\u52a0\u3067\u30b3\u30d4\u30fc\u3059\u308b\u
 ReplaceOperation.DisplayName=\u6587\u5b57\u5217\u3092\u7f6e\u304d\u63db\u3048\u308b
 # EnableOperation.DisplayName=ジョブを有効にする
 EnableOperation.DisplayName=\u30b8\u30e7\u30d6\u3092\u6709\u52b9\u306b\u3059\u308b
+# DisableOperation.DisplayName=��
+DisableOperation.DisplayName=
```

Comment:
JFYI: You can just leave Messages.properties for unknown languages unchanged.
Jenkins displays default (English in this case) texts for texts without translations, and provides a tool to look up missing translations. Adding invalid translations disturb that.
Please have a look at https://wiki.jenkins-ci.org/display/JENKINS/Translation+Tool for more information.

(You don't need further work for this pull request as I'll create another commit to add Japanese translation.)